proposals/description/fip_53.ts

```diff
@@ -0,0 +1,63 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_53: ProposalDescription = {
+  title: 'FIP-53: Farm d3Pool on Convex',
+  commands: [
+    {
+      target: 'fei',
+      values: '0',
+      method: 'mint(address,uint256)',
+      arguments: ['{feiDAOTimelock}', '50000000000000000000000000'],
+      description: 'Mint 50M FEI for liquidity deposit'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'approve(address,uint256)',
+      arguments: ['{curveD3pool}', '50000000000000000000000000'],
+      description: 'Approve 50M FEI for deposit on Curve d3pool'
+    },
+    {
+      target: 'curveD3pool',
+      values: '0',
+      method: 'add_liquidity(uint256[3],uint256)',
+      arguments: [
+        [
+          '0', // 0 FRAX
+          '50000000000000000000000000', // 50M FEI
+          '0' // 0 alUSD
+        ],
+        // we tolerate 0.5% slippage, set this value in Curve's UI and note the
+        // minimum amount of LP tokens out https://curve.fi/factory/57/deposit
+        '49580000000000000000000000' // min LP out
+      ],
+      description: 'Deposit 50M FEI in Curve to get LP tokens'
+    },
+    {
+      target: 'curveD3pool',
+      values: '0',
+      method: 'approve(address,uint256)',
+      arguments: ['{convexBooster}', '49580000000000000000000000'],
+      description: 'Approve d3pool LP tokens to on Convex'
+    },
+    {
+      target: 'convexBooster',
+      values: '0',
+      method: 'deposit(uint256,uint256,bool)',
+      arguments: ['58', '49580000000000000000000000', true], // Convex d3pool id = 58
+      description: 'Stake d3pool LP tokens on Convex'
```

Comment:
This proposal is incomplete. It should also deploy a new PCVDeposit to add this 50m FEI to protocol controlled FEI and add that PCVDeposit to the CR Oracle.

Reply:
Added a new PCVDeposit to do it, and updated the proposal to track the new deposit through CR Oracle